<!-- ================= Jetson1 ======================= -->
<node pkg="wp3_decompressor" type="wp3_decompressor_node" name="jetson1_decomp_kinect">
    <param name="sensor_name" value="jetson1" />
    <param name="sensor_type" value="0" />
    <param name="filter_value" value="1" />
</node>

<!-- ================= Jetson2 ======================= -->
<node pkg="wp3_decompressor" type="wp3_decompressor_node" name="jetson2_decomp_kinect">
    <param name="sensor_name" value="jetson2" />
    <param name="sensor_type" value="0" />
    <param name="filter_value" value="1" />
</node>

<!-- ================= Jetson3 ======================= -->
<node pkg="wp3_decompressor" type="wp3_decompressor_node" name="jetson3_decomp_kinect">
    <param name="sensor_name" value="jetson3" />
    <param name="sensor_type" value="0" />
    <param name="filter_value" value="1" />
</node>

<!-- ================= Jetson4 ======================= -->
<node pkg="wp3_decompressor" type="wp3_decompressor_node" name="jetson4_decomp_kinect">
    <param name="sensor_name" value="jetson4" />
    <param name="sensor_type" value="0" />
    <param name="filter_value" value="1" />
</node>

<!-- ================= Jetson5 ======================= -->
<node pkg="wp3_decompressor" type="wp3_decompressor_node" name="jetson5_decomp_kinect">
    <param name="sensor_name" value="jetson5" />
    <param name="sensor_type" value="0" />
    <param name="filter_value" value="1" />
</node>

<!-- ================= Jetson6 KINECT ======================= -->
<node pkg="wp3_decompressor" type="wp3_decompressor_node" name="jetson6_decomp_kinect">
    <param name="sensor_name" value="jetson6" />
    <param name="sensor_type" value="0" />
    <param name="filter_value" value="1" />
</node>
<!-- ================= Jetson6 VELODYNE ======================= -->
<node pkg="wp3_decompressor" type="wp3_decompressor_node" name="jetson6_decomp_velodyne">
    <param name="sensor_name" value="jetson6" />
    <param name="sensor_type" value="1" />
    <param name="filter_value" value="1" />
</node>
